### PR TITLE
Auto-generate Monitoring for Python

### PR DESCRIPTION
**Pull request by bjwatson**

### CODE REVIEW

File: generated/python/gapic-google-monitoring-v3/google/cloud/gapic/monitoring/v3/group_service_api.py [hunk 1]
```diff
@@ -0,0 +1,458 @@
...
+        Raises:
+          :exc:`google.gax.errors.GaxError` if the RPC is aborted.
+          :exc:`ValueError` if the parameters are invalid.
+        """
+        request = group_service_pb2.DeleteGroupRequest(name=name)
+        self._delete_group(request, options)
+
+    def list_group_members(self,
+                           name,
+                           filter_,
```
Comment (geigerj) [inline generated/python/gapic-google-monitoring-v3/google/cloud/gapic/monitoring/v3/group_service_api.py]: optional

File: generated/python/gapic-google-monitoring-v3/google/cloud/gapic/monitoring/v3/group_service_api.py [hunk 2]
```diff
@@ -0,0 +1,458 @@
...
+                           interval,
```
Comment (geigerj) [inline generated/python/gapic-google-monitoring-v3/google/cloud/gapic/monitoring/v3/group_service_api.py]: optional

(please double-check other methods, too.)

File: generated/python/gapic-google-monitoring-v3/google/cloud/gapic/monitoring/v3/metric_service_api.py [hunk 3]
```diff
@@ -0,0 +1,607 @@
...
+            self.metric_service_stub.ListTimeSeries,
+            settings=defaults['list_time_series'])
+        self._create_time_series = api_callable.create_api_call(
+            self.metric_service_stub.CreateTimeSeries,
+            settings=defaults['create_time_series'])
+
+    # Service calls
+    def list_monitored_resource_descriptors(self,
+                                            name,
+                                            filter_,
```
Comment (geigerj) [inline generated/python/gapic-google-monitoring-v3/google/cloud/gapic/monitoring/v3/metric_service_api.py]: optional

File: generated/python/gapic-google-monitoring-v3/google/cloud/gapic/monitoring/v3/metric_service_api.py [hunk 4]
```diff
@@ -0,0 +1,607 @@
...
+        Raises:
+          :exc:`google.gax.errors.GaxError` if the RPC is aborted.
+          :exc:`ValueError` if the parameters are invalid.
+        """
+        request = metric_service_pb2.DeleteMetricDescriptorRequest(name=name)
+        self._delete_metric_descriptor(request, options)
+
+    def list_time_series(self,
+                         name,
+                         filter_,
```
Comment (geigerj) [inline generated/python/gapic-google-monitoring-v3/google/cloud/gapic/monitoring/v3/metric_service_api.py]: `filter_`, `interval`, `order_by` optional

**Comment by bjwatson** [comment]
@geigerj Updated. PTAL

**Pull request merged**